src/Npgsql/TypeMapping/ConnectorTypeMapper.cs

```diff
@@ -93,7 +93,19 @@ internal bool TryResolveByOID(uint oid, [NotNullWhen(true)] out NpgsqlTypeHandle
 
         lock (_writeLock)
         {
-            if ((handler = ResolveByDataTypeName(pgType.SchemaQualifiedName, throwOnError: false)) is not null)
+            if ((handler = ResolveByDataTypeNameCore(pgType.FullName)) is not null)
```

Comment:
I'm not sure we really need this - will a resolver ever know about a specific type in a specific schema? Maybe only some special user-written one, tailored for a specific scenario or something...

Reply:
We probably don't need this, but it's definitely a nice thing to support just in case (if we do go through with resolver cleanup in 7.0, we'll essentially going to support this anyway, just at resolver level, and not at TypeMapper).